crates/rover-client/src/query/graph/push.rs

```diff
@@ -56,15 +56,15 @@ fn build_response(
 ) -> Result<PushResponse, RoverClientError> {
     if !push_response.success {
         let msg = format!("Schema upload failed with error: {}", push_response.message);
-        return Err(RoverClientError::HandleResponse { msg });
+        return Err(RoverClientError::AdhocError { msg });
     }
 
     let hash = match &push_response.tag {
         // we only want to print the first 6 chars of a hash
         Some(tag_data) => tag_data.schema.hash.clone()[..6].to_string(),
         None => {
             let msg = format!("No schema tag info available ({})", push_response.message);
```

Comment:
I know I wrote this error, but I hate it 😂 Nobody would know what a tag is
```suggestion
            let msg = format!("No data in response from schema push. Failed with message: {}", push_response.message);
```

Reply:
yeah... i was a bit confused about what it even did so i just left it haha